client.go

```diff
@@ -69,3 +79,19 @@ const (
 // Job is a function that will execute like a cron job after a
 // certain amount of time to perform some kind of task.
 type Job func(Bot, Channel) error
+
+//BroadcastFilter returns true if message can be broadcasted to bot and Channel
+type CanBeBroadcasted func(botID string, channel Channel, msg Sendable) bool
```

Comment:
i'd name this `BroadcastFilter` or `BroadcastFIlterFunc`. Also, why do you need to pass the message? You already have the message when you are calling Broadcast because you're sending it, right?